README.md

```diff
@@ -126,3 +126,72 @@ Now, for any new html page, we need to do the following:
 {% endblock %}
 
 ```
+
+## pre deployment to fly.io
+
+
+- Setting Up environmental variables
+
+  - Add `python-dotenv>=0.21.0` to requirements.txt
+  - Run `pip install -r requirements.txt`
+
+  -  create new .env file
+      add the following informations in it
+          SECRET_KEY
+          DATABASE_URL
+          DEBUG
```

Comment:
And what should the value be?

It would be much nicer if there were two separate and easy to use ways to:
1) Deploy to staging
2) Deploy to prod

The current README requires too much manual intervention and is too difficult to set up

Reply:
the SECRET_KEY and DEBUG can be changed on .env anytime for local development and there is conditional operator to use local or remote database on settings so. changing debug to false make it to run on local database

Reply:
Right, so make this as easy as possible for someone to set up staging and prod. Instead of English instructions, bash commands

Commands that you run only once, when you first check out the repo, and commands that you run repeatedly when the code is updated